video/out/vo_drm.c

```diff
@@ -213,6 +213,8 @@ static int reconfig(struct vo *vo, struct mp_image_params *params)
     if (mp_sws_reinit(p->sws) < 0)
         return -1;
 
+    // essentially constant, so this is okay
+    vo->target_params = &p->sws->dst;
```

Comment:
For consistency I would add params lock around those. Even tho during reconfig nothing should use it directly